assets/source/setup-guide/app/steps/SetupAccount.js

```diff
@@ -137,6 +138,20 @@ const SetupAccount = ( {
 										}
 										target="_blank"
 										rel="noreferrer"
+										onClick={ () =>
+											recordEvent(
+												'pfw_documentation_link_click',
+												{
+													link_id: 'ad-guidelines',
+													context: 'setup-account',
+													href:
+														wcSettings
+															.pinterest_for_woocommerce
+															.pinterestLinks
+															.adGuidelines,
+												}
+											)
+										}
```

Comment:
This PR is targetting the feature branch https://github.com/woocommerce/pinterest-for-woocommerce/tree/feature/232-jsdoc-tweaks/ I guess it would be cleaner to add that event here (or in a following PR) once https://github.com/woocommerce/pinterest-for-woocommerce/tree/feature/210-settings-pinterest-merchant-guidelines is merged to `develop`, then `develop` to here

Reply:
That's a better and thorough way. 👍  Thanks!